test/protractor.conf.js

```diff
@@ -89,7 +89,9 @@ exports.config = {
   // run.
   suites: {
     // smoke: 'spec/smoketests/*.js',
-    rest_api: 'integration/rest_api/*.js' // This suite of tests should only require a running master api, it should not require a node
+    'add-to-project': 'integration/features/user_adds_template_to_project.spec.js',
+    'create-project': 'integration/features/user_creates_project.spec.js' // This suite of tests should only require a running master api, it should not require a node
```

Comment:
I think it can be an array of strings.  I think we can do both, but its really nice (and saves a lot of dev time) if you can pluck out one test flow at a time.  I'm totally open to this growing/shrinking/shifting as we add tests.